toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -145,30 +178,51 @@ func (im *IsoMaker) buildIsoImage() {
 		// Directory to convert to an ISO.
 		im.buildDirPath)
 
-	shell.MustExecuteLive("mkisofs", mkisofsArgs...)
+	err := shell.ExecuteLive(false /*squashErrors*/, "mkisofs", mkisofsArgs...)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
If you'd like, you can also do this for such simpler cases:
```suggestion
	return shell.ExecuteLive(false /*squashErrors*/, "mkisofs", mkisofsArgs...)
```

Reply:
updated functions that have this pattern.